federatedscope/cl/trainer/trainer.py

```diff
@@ -0,0 +1,99 @@
+from federatedscope.core.auxiliaries.enums import MODE
+from federatedscope.register import register_trainer
+from federatedscope.core.auxiliaries.optimizer_builder import get_optimizer
+from federatedscope.core.auxiliaries.scheduler_builder import get_scheduler
+from federatedscope.core.trainers import GeneralTorchTrainer
+from federatedscope.core.trainers.context import CtxVar
+from federatedscope.core.auxiliaries.enums import LIFECYCLE
+from federatedscope.core.auxiliaries import utils
+import torch
+import numpy as np
+
+
+class CLTrainer(GeneralTorchTrainer):
```

Comment:
To be honest, `CL` is a very confusing name to me.

Reply:
it means Contrast Learning